neo/test/iotest/test_phyio.py

```diff
@@ -0,0 +1,94 @@
+"""
+Tests of neo.io.phyio
+"""
+
+import unittest
+
+from neo.io.phyio import PhyIO  # , HAVE_SCIPY
+from neo.test.iotest.common_io_test import BaseTestIO
+from neo.io.proxyobjects import (AnalogSignalProxy,
+                                 SpikeTrainProxy, EventProxy, EpochProxy)
+from neo import (AnalogSignal, SpikeTrain)
+
+import quantities as pq
+import numpy as np
+
+import tempfile
+from pathlib import Path
+
+
+class TestPhyIO(BaseTestIO, unittest.TestCase):
+    ioclass = PhyIO
+    files_to_test = ['phy_example_0']
+    files_to_download = [
+        'phy_example_0/spike_times.npy',
+        'phy_example_0/spike_templates.npy',
+        'phy_example_0/spike_clusters.npy',
+        'phy_example_0/params.py',
+        'phy_example_0/cluster_KSLabel.tsv',
+        'phy_example_0/cluster_ContamPct.tsv',
+        'phy_example_0/cluster_Amplitude.tsv',
+        'phy_example_0/cluster_group.tsv'
+    ]
+
+
+class SpecificTestPhyIO(unittest.TestCase):
+    def setUp(self):
+        temp_directory = Path(tempfile.gettempdir())
+        self.temp_folder = temp_directory.joinpath(
+            'files_for_testing_neo/phy/phy_example_0')
+
```

Comment:
To avoid circle ci fails.
You should not use another class but extend the TestPhyIO class.
So you could remove this setUp func and do instaed this:

```python
    def test_read_segment_lazy(self):
        dirname = self.get_filename_path('phy_example_0')
        r = PhyIO(dirname=self.temp_folder)
```

Tell me if it helps.

Reply:
This has helped, thanks! I have extended the TestPhyIO class and removed ```'seg_extra_info' in seg.annotations``` assertion. Now the Circle CI passes.